src/Raven.Server/RavenServer.cs

```diff
@@ -1591,11 +1734,15 @@ private bool TryAuthorize(RavenConfiguration configuration, Stream stream, TcpCo
                             }
                             if (auth.CanAccess(header.DatabaseName, requireAdmin: false))
                                 return true;
-                            msg = "The certificate " + certificate.FriendlyName + " does not allow access to " + header.DatabaseName;
+                            msg = "The certificate " + certificate?.FriendlyName + " does not allow access to " + header.DatabaseName;
                             return false;
                         default:
                             throw new InvalidOperationException("Unknown operation " + header.Operation);
                     }
+                case AuthenticationStatus.UnfamiliarIssuer:
+                    msg = $"The client certificate {certificate?.FriendlyName} is not registered in the cluster explicitly but is trusted implicitly by its Public Key Pinning Hash. " +
```

Comment:
It still isn't trusted. It _could_ be, but need the additional step.
Alternatively, let's tell the admin to just register the new cert

Reply:
Yes